examples/safe/src/account.js

```diff
@@ -0,0 +1,31 @@
+import { Safe4337Pack } from '@safe-global/relay-kit'
+import { privateKeyToAccount } from 'viem/accounts'
+import config from '../../../config.js'
+
+// To deploy a new Safe Smart Account
+const signer = privateKeyToAccount(config.private_key)
+const safeAccountOptions = {
+  owners: [signer.address],
+  threshold: 1
+}
+
+// To use a Safe Smart Account that already exists
+// const safeAccountOptions = {
+//   safeAddress: '0x...'
+// }
+
+export const account = await Safe4337Pack.init({
+  provider: config.rpc_url,
+  signer: config.private_key,
+  rpcUrl: config.rpc_url,
+  bundlerUrl: config.rpc_url,
```

Comment:
This should be the bundler URL and not a RPC URL

Reply:
It is the bundler URL: ```https://api.developer.coinbase.com/rpc/v1/base-sepolia/JOAtWTaE8FyhL5PL9txI8nIf4rb-lHN6```